docs/extend/extend-cli/cli-extending-a-plugin.md

```diff
@@ -93,91 +102,114 @@ const session = new Session({ hostname: "jsonplaceholder.typicode.com"});
 })();
 ```
 
-### Checkpoint one
+### Verify your build with the programmatic API **[correct?]**
```

Comment:
@adam-wolfe please respond

Reply:
I would change to either:
"Verify that the project compiles without errors"
or
"Build the project" 
or maybe even "Checkpoint: Build the project" since it is kind of a point in the tutorial where somebody would check to make sure that they haven't broken something, and the project can be compiled without errors.

Reply:
discussed in person